rbi/yast/builtins.rbi

```diff
@@ -0,0 +1,61 @@
+# typed: strong
+
+module Yast::Builtins
```

Comment:
Hi interesting, code few questions: Is there check that verify that rbi and real rb code matches?

Reply:
Good point!

- If I make a mistake in builtins.rbi, it will go undetected because builtins.rb is still `typed: false`
- convert.rb is `typed: true` but it defines its methods with metaprogramming so again, undetected :(
- ops.rb is `typed: false` but if I change it to `true` there are few enough errors and one of them is in fact a mismatch that you ask about:

```
src/ruby/yast/ops.rb:113: Method Yast::Ops.get redefined without matching argument count. Expected: 3, got: 4 https://srb.help/4010
     113 |    def self.get(object, indexes, default = nil, skip_frames = 0)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    rbi/yast/ops.rbi:15: Previous definition
    15 |  def self.get(object, indexes, default=T.unsafe(nil), &block); end
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```